LEGO1/mxdsstreamingaction.cpp

```diff
@@ -0,0 +1,91 @@
+#include "mxdsstreamingaction.h"
+
+DECOMP_SIZE_ASSERT(MxDSStreamingAction, 0xb4)
+
+// OFFSET: LEGO1 0x100cd010
+MxDSStreamingAction::MxDSStreamingAction(MxDSAction &p_dsAction, MxU32 p_offset)
+{
+  Init();
+
+  MxDSAction::operator=(p_dsAction);
```

Comment:
This is interesting, I feel like what this might actually resolve to is `*this = p_dsAction`. I _believe_ that would produce the same result but is more likely to be what was written by the developers.

Reply:
Hmm if it doesn't work out of the box than maybe not? It just seems like an odd thing for the developers to have written explicitly, but I guess the asm doesn't lie. Up to you if you want to keep the change or not.

Reply:
I'd keep the change, it makes the code a little more readable and isn't completely illogical. Welcoming any suggestions for further improvements in the future of course.

Reply:
Sounds good, will merge